internal/clients/ec2/ec2_client.go

```diff
@@ -120,3 +120,34 @@ func (c *Client) ListInstanceTypes() ([]types.InstanceTypeInfo, error) {
 
 	return resp.InstanceTypes, nil
 }
+
+func (c *Client) RunInstances(ctx context.Context, amount int32, instanceType types.InstanceType, AMI string, keyName string) ([]*string, *string, error) {
+	log.Trace().Msg("Run AWS EC2 instance")
+	input := &ec2.RunInstancesInput{
+		MaxCount:     aws.Int32(amount),
+		MinCount:     aws.Int32(amount),
+		InstanceType: instanceType,
+		ImageId:      aws.String(AMI),
+		KeyName:      &keyName,
+	}
+	resp, err := c.ec2.RunInstances(ctx, input, func(o *ec2.Options) {
+		o.Region = config.AWS.Region
```

Comment:
does this do something? aren't we initializing the client with region already?
I know we will probably need it in the future, but now it just looks funny to override one static value with the same one.

Reply:
yes you are right, it is not needed :)